scripts/User_Management/grade_9_mode.py

```diff
@@ -0,0 +1,41 @@
+from scripts._utils import ssh, pi, utils
+
+host_themes = "pi-themes"
+tvs = ["1", "2", "3", "4"]
+
+TV_OFF_CMD = "echo standby 0 | cec-client -s -d 1"
+TV_ON_CMD = "echo on 0 | cec-client -s -d 1"
+
+
+def engage():
+    ssh_connection = ssh.SSH(host_themes, pi.username, pi.password)
+    ssh_connection.send_cmd("amixer sset 'Headphone',0 0%")
+
+    for tv_number in tvs:
+        hostname = "pi-tv{}".format(tv_number)
+        ssh_connection = ssh.SSH(hostname, pi.username, pi.password)
+
+        ssh_connection.send_cmd(TV_OFF_CMD)
+        ssh_connection.close()
+
+
+def disengage():
+    ssh_connection = ssh.SSH(host_themes, pi.username, pi.password)
+    ssh_connection.send_cmd("amixer sset 'Headphone',0 100%")
+
+    for tv_number in tvs:
+        hostname = "pi-tv{}".format(tv_number)
+        ssh_connection = ssh.SSH(hostname, pi.username, pi.password)
+
+        ssh_connection.send_cmd(TV_ON_CMD)
+        ssh_connection.close()
+
+
+def grade_9_mode():
+    confirmation = utils.input_styled("Do you want to engage grade 9s? [y/N] (q to quit): \n")
```

Comment:
This is ambiguous, and it's not clear from prompt the difference between N and q. Should ask to [e]ngage, [D]isengage, or [q]uit.

Should also have a description of what it does:
" Enganging grade 9 mode will:
   * Thing 1
   * Thing 2
   * etc.
"

Reply:
fixed as per [6612eae](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/60/commits/6612eae166341b92fd6439632f661f323301261d) and [8bec1cf](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/60/commits/8bec1cfe789ef129a71e89d962594cdf4165f7fe)